### ISSUE TITLE
Npm install error with @shoutem/theme dependency

### ISSUE THREAD

**Issue by MathiasMeuleman** [GeekyAnts/NativeBase#716]
Hi,

I'm trying to install native-base via npm on my Windows machine. When installing, I get the following error message:

npm install native-base --save

[native-base-install-error.txt](https://github.com/GeekyAnts/NativeBase/files/895905/native-base-install-error.txt)

The debug-log, just to be sure:

[2017-04-05T10_18_04_990Z-debug.txt](https://github.com/GeekyAnts/NativeBase/files/895909/2017-04-05T10_18_04_990Z-debug.txt)

It seems to be a problem with the @shoutem/theme dependency that can't be aquired with the url you are using in package.json. Now I tried installing @shoutem/theme manually via npm, everything worked fine:

npm install @shoutem/theme --save

So why aren't you simply adding @shoutem/theme with a version, instead of the git url?

Thanks in advance!

current versions:
node: v7.8.0
npm: v4.4.4
react-native-cli: v2.0.1
react-native: v0.41.2

**Comment by MathiasMeuleman** [comment]
I managed to do a work-around by downloading your sourcecode, manually changing the url with the version number (currently 0.9.0) and installing that package via npm. Still I am curious to the reason you made this choice.

**Comment by sankhadeeproy007** [comment]
cc @himanshu-satija

**Comment by himanshu-satija** [comment]
We have forked our own copy of @shoutem/theme because we needed to customize it according to our requirements. If you have installed @shoutem/theme directly from npm, you might experience some issues. Mostly the performance will be affected since we have implemented caching in our version of @shoutem/theme.

When native-base is installed, it's @shoutem/theme dependency gets installed from our GitHub repo. We know this is not ideal. Soon we are going to publish our own version of @shoutem/theme on npm so that dependency management becomes simple.
